nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfig.js

```diff
@@ -391,3 +368,40 @@ function GatewayRAN({gwInfo}: {gwInfo: lte_gateway}) {
 
   return <DataGrid data={ran} />;
 }
+
+function ApnResourcesTable({gwInfo}: {gwInfo: lte_gateway}) {
+  const apnResources = gwInfo.apn_resources || {};
+  type ApnResourcesRowType = {
+    name: string,
+    id: string,
+    vlanId: number | string,
+  };
+  const apnResourcesRows: Array<ApnResourcesRowType> = Object.keys(
+    apnResources,
+  ).map((apn: string) => {
+    const apnRow = apnResources[apn];
+    return {
+      name: apn,
+      id: apnRow.id,
+      vlanId: apnRow.vlan_id ?? '-',
+    };
+  });
+
+  return (
+    <ActionTable
+      title=""
+      data={apnResourcesRows}
+      columns={[
+        {title: 'Name', field: 'name'},
+        {title: 'Resource ID', field: 'id'},
+        {title: 'VLAN ID', field: 'vlanId'},
+      ]}
+      options={{
+        actionsColumnIndex: -1,
+        pageSizeOptions: [5],
+        toolbar: false,
+        paging: false,
+      }}
```

Comment:
Would this not display all the APN resources if there were more than 5?

Reply:
Oh you're right I need to set paging: true